internal/store/ingress.go

```diff
@@ -156,9 +156,13 @@ func ingressMetricFamilies(allowAnnotationsList, allowLabelsList []string) []gen
 									Value:       1,
 								})
 							} else {
+								apiGroup := ""
+								if path.Backend.Resource.APIGroup != nil {
+									apiGroup = *path.Backend.Resource.APIGroup
+								}
 								ms = append(ms, &metric.Metric{
-									LabelKeys:   []string{"host", "path", "service_name", "service_port"},
-									LabelValues: []string{rule.Host, path.Path, "", ""},
+									LabelKeys:   []string{"host", "path", "api_group", "kind", "name"},
```

Comment:
```suggestion
									LabelKeys:   []string{"host", "path", "resource_api_group", "resource_kind", "resource_name"},
```

Reply:
or we could use `backend` as a prefix, I don't have a preference, but we need to scope the labels IMO